activerecord/lib/active_record/tasks/database_tasks.rb

```diff
@@ -388,6 +392,21 @@ def each_local_configuration
         def local_database?(configuration)
           configuration["host"].blank? || LOCAL_HOSTS.include?(configuration["host"])
         end
+
+        def structure_version_dump(structure_filename)
+          return unless SchemaMigration.table_exists?
+          version_filename = structure_filename.sub(".sql", ".version")
+          version = Base.connection.migration_context.current_version
+          File.write(version_filename, version)
+        end
+
+        def structure_version_load(structure_filename)
+          return unless SchemaMigration.table_exists?
+          version_filename = structure_filename.sub(".sql", ".version")
+          return unless File.exists?(version_filename)
+          version = File.read(version_filename)
+          Base.connection.assume_migrated_upto_version(version, [])
```

Comment:
I think the second arg of this method isn't used anymore. Is this public API or can we just remove it?

Reply:
This is technically public (search `assume_migrated_upto_version` on https://api.rubyonrails.org/) but you're right it looks like it's no longer used by that method. I think we can safely delete it.